spring_boot_interview/src/test/java/com/luishipero/interview/InterviewApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.luishipero.interview;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class InterviewApplicationTests {
```

Comment:
Hi Ricardo, no special reason. All comes down to lack of time and because I wanted to meet the deadline it was given.

I've pushed some test code, so you can have a look on how I would be implementing tests.